python/Ganga/testlib/GangaUnitTest.py

```diff
@@ -48,6 +48,7 @@ def start_ganga(gangadir_for_test, extra_opts=[]):
         ('Configuration', 'gangadir', gangadir_for_test),
         ('Configuration', 'user', 'testframework'),
         ('Configuration', 'repositorytype', 'LocalXML'),
+        ('Configuration', 'UsageMonitoringMSG', False),  # Turn off spyware
```

Comment:
1. It turns out that `dashb-mb-test.cern.ch` doesn't exist any more
2. Setting it to an non-existent server makes a test of the submission of a simple local job go from 5 seconds to 15 seconds due to it trying to establish a connection.

```
WARNING  Could not connect to host dashb-mb-test.cern.ch, port 61113: [Errno -2] Name or service not known
```